code/game/objects/items/devices/radio/radio.dm

```diff
@@ -116,6 +116,11 @@
 
 	AddElement(/datum/element/empprotection, EMP_PROTECT_WIRES)
 
+	var/static/list/loc_connections = list(
+		COMSIG_RADIO_JAMMED = PROC_REF(on_jammed),
+	)
+	AddElement(/datum/element/connect_loc, loc_connections)
```

Comment:
This doesn't work, only items directly worn on your person will be jammed. If you have it in your bag, or you try to jam the radio itself (i.e. click on an intercom, or a handheld radio dropped on the ground, etc) nothing will happen.

Reply:
Do we have any other alternatives we can use?

I was thinking of making ``connect_loc`` be recursive but it would require registering signal on every level of content it's in up to the mob, then unregistering from all of them when moved, plus would have to register move for the contents it's in as well.

Now I'm thinking of registering the signal directly on radio's ``equip()`` to the mob picking it up, but I think that would also not work if you put it in your bag?

Reply:
We have a component, `connect_containers`, which would service what you're looking for

Reply:
the goat